apps/api/lib/email/templates/validate-email-address-successful-mentee.mjml

```diff
@@ -49,14 +46,14 @@
             css-class="button-mobile"
             href="https://www.youtube.com/watch?v=M3nwS3QfdMM"
           />
-          <mj-text mj-class="text" padding="0 0 20px 0"
+          <mj-text mj-class="text" padding="0 0 20px 0" align="justify"
             >Your <strong>final step</strong> after watching this video will be
             your <strong>activation call</strong> with our Mentorship Community
-            Manager Johanna, who will be happy to answer any remaining questions
-            and will then activate your account - only then will you be able to
+            Manager, <strong>Hadeer</strong>, who will be happy to answer any remaining questions
+            and then activate your account - only then will you be able to
             fully use the platform and search for a mentor.</mj-text
           >
-          <mj-text mj-class="text" padding="0 0 0 0"
```

Comment:
I assume removing `padding="0 0 0 0"` - as it's been done in many places - has no effect (or negligible) on layout?

Reply:
I use MJML Preview in VS Code to view the changes in the emails. I didn't see any effect of `padding="0 0 0 0"` (neither in desktop or mobile view), that's why I chose to remove it. I didn't do this in all the emails, though.

Reply:
Can you trigger a "real" email and check it in your browser, to double check? I think the MJML Preview tool does a decent job, but I've found a need to do actual tests to verify.

Interested in hearing back on results though - if there is a high one-to-one similarity between the preview and the real deal. Maybe we can start trusting the preview more.

Reply:
I did a "real" e-mail check as well. Not for every changed e-mail yet, though. I planned to do a "real" test for all after PR approval:)

Reply:
This is a result in a "real" email:
![image](https://user-images.githubusercontent.com/51786805/201360738-debe10b1-5bd5-45b7-810e-9ee2aa566f52.png)

Reply:
Perfect. Then we can be reasonably confident that the removed `padding="0 0 0 0"` will have a similar effect for all other affected emails.